aea/configurations/data_types.py

```diff
@@ -86,6 +86,8 @@ class PackageVersion:
 
     _version: PackageVersionLike
 
+    __slots__ = ("_version",)
+
```

Comment:
such that a generic solution could be implemented that is equal for all classes in this module